.github/workflows/reusable-unit.yml

```diff
@@ -128,9 +128,13 @@ jobs:
         run: yarn ${{ matrix.package }} build
 
       - name: Run ${{ matrix.package }} package unit tests
+        # TODO no-lights remove once new sdk is set up  
```

Comment:
Can we make an asana task to come back to this, or add it as a requirement to our e2e test task?

Reply:
yes!